lib/src/ast/element.dart

```diff
@@ -9,7 +9,7 @@ import 'package:collection/collection.dart';
 import 'package:source_span/source_span.dart';
 import 'package:quiver/core.dart';
 
-const _listEquals = const ListEquality();
+const _listEquals = const ListEquality<dynamic>();
```

Comment:
Not necessarily, but the IDE was complaining about it not having a type defined.